### PR TITLE
Clocker

### PR DESCRIPTION
**Pull request by n0shake**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Welcome, Dr. Doofenshmirtz, and thank you for the submission. In the future, please use `x` in checklists (no spaces inside the brackets) and [Github will make them clickable](https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments).

File: Casks/clocker.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'clocker' do
+  version '1.5.1'
+  sha256 'd9003079b1c2451932281afab9742943897b0b4015181cc7e1ed1c6c3651db84'
+
+  # github.com/abhishekbanthia/Clocker was verified as official when first introduced to the cask
```
Comment (vitorgalvao) [inline Casks/clocker.rb]: Comment needed to follow the pattern.

File: Casks/clocker.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  url "https://github.com/abhishekbanthia/Clocker/releases/download/v#{version}/Clocker.zip"
+  name 'Clocker'
```
Comment (vitorgalvao) [inline Casks/clocker.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/clocker.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  homepage 'https://abhishekbanthia.com/Clocker'
+
+  app 'Clocker.app'
+end
```
Comment (vitorgalvao) [inline Casks/clocker.rb]: Please add a [`zap`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md) as well.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by abhishekbanthia** [comment]
Thanks for the quick review. I've added appcast and zap. Let me know if I'm missing anything.

**Pull request merged**